lib/capistrano/multiconfig.rb

```diff
@@ -1,5 +1,8 @@
 require 'capistrano/multiconfig/dsl'
 
+require "capistrano/doctor"
```

Comment:
Why doctor should be required by multiconfig?
It does not use it.
If you want it you may to add to your application in `Capfile`